src/main/java/com/ty/workSpace_Management/repo/EmployeeRepo.java

```diff
@@ -0,0 +1,21 @@
+package com.ty.workSpace_Management.repo;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import com.ty.workSpace_Management.entity.EmployeeEntity;
+
+public interface EmployeeRepo extends JpaRepository<EmployeeEntity, String> {
+	
+	@Query(" select e from EmployeeEntity e where e.employeeEmail = ?1 and e.employeePassword = ?2 ")
```

Comment:
pls provide the login method in service layer 
because we need to throw the exception based on email and pwd